lib/web/apiserver_test.go

```diff
@@ -1035,6 +1041,35 @@ func (s *WebSuite) TestResizeTerminal(c *C) {
 	}
 }
 
+func (s *WebSuite) TestTerminalPing(c *C) {
+	ws, err := s.makeTerminal(s.authPack(c, "foo"))
+	c.Assert(err, IsNil)
+	defer ws.Close()
+	closed := false
+
+	done := make(chan struct{})
```

Comment:
nit: move variables that work together closer together?

```suggestion

	closed := false
	done := make(chan struct{})
```